CapstoneProject/Assets/Scripts/Core/Main Menu/MainMenu.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenu : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public void Continue()
+    {
+        SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex + 1);
```

Comment:
you should probably have an enum for the scenes or something instead of +1 and +2